repos/system_upgrade/el8toel9/actors/multipathconfupdate/tests/test_multipath_conf_update_8to9.py

```diff
@@ -0,0 +1,176 @@
+import os
+
+import pytest
+
+from leapp.libraries.actor import multipathconfupdate
+from leapp.libraries.common import multipathutil
+from leapp.models import MultipathConfFacts8to9, MultipathConfig8to9
+
+BEFORE_DIR = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'files/before')
+AFTER_DIR = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'files/after')
+
+converted_data = {}
+
+
+def build_config(val):
```

Comment:
Same suggestion as above - 5 distinct arguments for this helper.

Reply:
Yep